### PR TITLE
skip global hub test for doc/samples updates

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml [hunk 1]
```diff
@@ -74,8 +74,9 @@ tests:
     make unit-tests
   container:
     from: src
+  skip_if_only_changed: ^doc/|samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
 - as: test-e2e
-  skip_if_only_changed: ^doc/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
+  skip_if_only_changed: ^doc/|samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml:79]: the e2e test depends the kafka and postgres samples. if you remove it, we cannot verify the changes in samples.

Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml:79]: @clyang82 According to my check, I didn't find any files in e2e that reference samples inside.
Instead, there are some files in `operator/config/samples` that are used by the e2e test.

Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml]: Only skip the `^sample`

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**